src/Robo/Plugin/Commands/CICommands.php

```diff
@@ -52,7 +52,7 @@ public function __construct()
         $this->phpcsCheckExtensions = Robo::config()->get('phpcs_check_extensions');
         $this->phpcsIgnorePaths = Robo::config()->get('phpcs_ignore_paths');
         $this->customCodePaths = implode(' ', $this->getCustomCodePaths());
-        $this->lintTwigFiles = Robo::config()->get('twig_lint_enable', true);
+        $this->lintTwigFiles = Robo::config()->get('twig_lint_enable') ?? true;
```

Comment:
Interesting change from relying on the `get()` method to relying on php itself. I would love to hear your thoughts on the change and why you think it is better to provide the default via php rather than using the `get()` methods provision for that. I have no opinion on the matter.

Reply:
I think the issue was that the second (default) parameter was typed as a `string` and it didn't like that I was specifying a `bool`, but I _wanted_ a `bool`, so this was my solution.

Reply:
Makes perfect sense.

Reply:
💯 on the question and the answer. I'm sold.